### PR TITLE
Update social_contract.md

### PR DESCRIPTION
**Pull request by ericlp**

### CODE REVIEW

File: documents/social_contract.md [hunk 1]
```diff
@@ -15,7 +15,9 @@ The communication in the group is done through Slack and through the meetings of
 
 ## Deadlines
 
-All turn-ins (Group and individual) is turned in on Thursdays. The SCRUM master will make sure that the members have turned in their parts.
+All turn-ins (Group and individual) is turned in on Thursdays.
```
Comment (ViddeM) [inline documents/social_contract.md]: Should this not be on mondays/sundays?

**Review by ViddeM — DISMISSED** [review]

File: documents/social_contract.md (hunk 1 above)
Comment (ericlp) [reply documents/social_contract.md]: It felt like we had yet to decide as a group whether or not we should move the individual reflection to mondays as well.

Comment (PontusLindblom) [reply documents/social_contract.md]: I felt like we agreed on doing both the group and individual discussion on Mondays

**Review by PontusLindblom — APPROVED** [review]

**Comment by PontusLindblom** [comment]
I re-ran Travis so that the build worked. Now we only need more reviews

**Review by ghost — APPROVED** [review]

**Review by ejohnsson6 — APPROVED** [review]

**Pull request merged**